src/sass/components/_feedback.scss

```diff
@@ -0,0 +1,144 @@
+.container {
+  position: relative;
+
+  .row {
+    &.overwritten {
+      max-width: 90%;
+    }
+  }
+
+  .carousel {
+    width: 90%;
+    margin: 20px auto 0 auto;
+
+    .carousel-cell {
+      width: 100%;
+      display: flex;
+      flex-direction: column;
+      justify-content: center;
+
+      .fa-quote-right {
+        width: 23px;
+        height: 19px;
+        color: #2a2a2a;
+        margin: 12px auto 30px auto;
+      }
+
+      p {
+        width: 90%;
+        text-align: right;
+        color: $post-info-text-color;
+        font-size: 14px;
+        font-weight: 400;
+        line-height: 25px;
+      }
+
+      .client {
+        margin: 10px auto 61px auto;
+        height: 80px;
+        width: 400px;
+        display: flex;
+        align-items: center;
+
+        .avatar {
+          width: 60px;
+          height: 52px;
+          border-radius: 5px;
+          border: 1px solid #aaaaaa;
+          background-color: $post-info-text-color;
+          justify-content: flex-start;
+        }
+
+        .client-info {
+          display: flex;
+          flex-direction: column;
+          width: auto;
+          max-height: 60px;
+          max-width: 250px;
+
+          .name {
+            color: $post-info-text-color;
+            font-family: "Poppins - Semi Bold";
```

Comment:
Taki zapis `Poppins - Semi Bold` jest poprawny? Nie mogę wyszukać takiego zapisu.
Dalej masz `font-weight: 600` ustawione co jest jednoznaczne z **Poppins Semi Bold** z dokumentacji google fonts, a **Poppins** to nasz bazowy font :)

Chyba samo `font-weight` wystarczy

Reply:
Poprawione :)